Find.md

```diff
@@ -0,0 +1,202 @@
+# JULEP find
+
+- **Title:** Reorganize Search and Find API
+- **Authors:** Milan Bouchet-Valat <<nalimilan@club.fr>>
+- **Created:** December 10, 2016
+- **Status:** work in progress
+
+## Abstract
+
+The current `find` and `search` families of functions are not very consistent as regards
```

Comment:
I'll come out in favor of "with regard to":
https://books.google.com/ngrams/graph?content=with+regards+to%2Cwith+regard+to%2Cas+regards&year_start=1800&year_end=2008&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cwith%20regards%20to%3B%2Cc0%3B.t1%3B%2Cwith%20regard%20to%3B%2Cc0%3B.t1%3B%2Cas%20regards%3B%2Cc0